src/tools/rest-backend/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+find_package (CppCMS)
+find_package (Boost 1.45 COMPONENTS system filesystem thread)
+find_package (LibJWT)
+find_package (OpenSSL)
+
+if (NOT CPPCMS_FOUND)
+    remove_tool (${tool} "CppCMS package not found")
+elseif (NOT Boost_FOUND)
+    remove_tool (${tool} "Boost package not found")
+elseif (NOT LIBJWT_FOUND)
+    remove_tool (${tool} "LibJWT package not found")
+elseif (NOT OPENSSL_FOUND)
+    remove_tool (${tool} "OpenSSL package not found")
+else ()
+
+    set (REST_AUTH_ENCRYPT_KEY "please_change_to_secret_key" CACHE STRING "Encryption key to be used for auth tokens")
+    set (REST_JWT_ISSUER "additional_secret_issuer" CACHE STRING "The issuer to be used for JWT tokens, should be kept secret")
+    set (REST_REPOSITORY_CONFIGS "dir/configs" CACHE STRING "Path to be used to store configuration snippets")
+    set (REST_REPOSITORY_USERS "dir/users" CACHE STRING "Path to be used to store user information")
```

Comment:
You are right, it would make more sense to have them run-time configured. Would it make sense to also add them to the configuration file provided by cppcms (they are tightly-coupled anyway)? Is it sufficient to mount the cppcms configuration or do you have something else in mind?